### PR TITLE
Send home button to nycmesh.net & remove NEXT_PUBLIC_MESHDB_URL

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Send the join form's "Go Home" button to nycmesh.net instead of forms.nycmesh.net, since it's unlikely join form submitters want to run the query form or NN assign forms

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,17 +16,17 @@ This repo currently powers three separate forms:
...
+   Then continue to the next section below and use the value `http://127.0.0.1:8000` for `MESHDB_URL`
 
 To get started with this package, first, create your own copy of the `.env` file:
 
 ```
 cp .env.sample .env
 ```
 
-Edit the `NEXT_PUBLIC_MESHDB_URL` based on your choice above in your favorite text editor. For example:
+Edit the `MESHDB_URL` and `NEXT_PUBLIC_MESHDB_URL` values based on your choice above in your favorite text editor. For example:
```
Comment (Andrew-Dickinson) [inline README.md:29]: @WillNilges why are there two separate variables now?

Comment (WillNilges) [reply README.md:29]: I think `NEXT_PUBLIC_MESHDB_URL` is vestigal and can be removed. NEXT_PUBLIC variables get compiled into the image so they can be accessed client side, which we can't do if we want to have a dev and prod enviornment.

I think you can just remove it, but grep through to make sure. `MESHDB_URL` is the thing to use.

File: components/JoinForm/JoinForm.tsx [hunk 2]
```diff
@@ -389,7 +389,12 @@ export default function App() {
           </p>
         </div>
         <div className={styles.centered} style={{ padding: "10px" }}>
-          <Button name="home" variant="contained" size="large" href="/">
+          <Button
+            name="home"
+            variant="contained"
+            size="large"
+            href="https://nycmesh.net/"
```
Comment (WillNilges) [inline components/JoinForm/JoinForm.tsx:396]: Ope yea good catch. Please update the tests.

**Review by WillNilges — APPROVED** [review]
Feel free to remove NEXT_PUBLIC_MESHDB_URL

**Review by WillNilges — APPROVED** [review]

**Pull request merged**